containers/daap-python-base/src/var/task/curated_data/curated_data_loader.py

```diff
@@ -80,3 +81,182 @@ def ingest_raw_data(self, raw_data_table: QueryTable, extraction_timestamp: str)
+        self.glue_client = glue_client
+        self.athena_client = athena_client
+        self.logger = logger
+        self.get_tables_to_copy()
+
+    def validate(self):
+        """
+        Check whether it is possible to copy the data of changed schema
+        between two major versions
+
+        Columns removed -> OK (will be ignored by athena)
+        Columns added -> OK (will be left as null)
+        Columns reordered -> OK
+        Data types changed -> Not OK
+        """
+
+        if self.column_changes["types_changed"]:
+            return False
+        else:
+            return True
+
+    def run(self):
+        """
+        This runs the actual copy of each data product table into the new database version
+
+        The table for where schema has been updated is created via boto3
+        as it will not always have data loaded to it via this method
+        """
+        # create new version database and table for updated schema
+        create_glue_database(self.glue_client, self.new_database_name, self.logger)
+        parquet_table_input = format_parquet_glue_table_input(
+            self.schema.data["TableInput"],
+            # os.path.join(self.new_curated_data_product_path, self.schema.table_name),
+            self.new_curated_data_product_path,
+        )
+        self.glue_client.create_table(
+            DatabaseName=self.new_database_name,
+            TableInput=parquet_table_input,
+        )
+
+        for table in self.tables_to_copy:
+            input_data = format_table_schema(
+                (
+                    DataProductSchema(
+                        data_product_name=self.data_product_name,
+                        table_name=table,
+                        logger=self.logger,
+                        input_data=None,
+                    )
+                    .load()
+                    .latest_version_saved_data
+                )
+            )
+            schema_for_copy = DataProductSchema(
+                data_product_name=self.data_product_name,
+                table_name=table,
+                logger=self.logger,
+                input_data=input_data,
+            )
+            schema_for_copy.convert_schema_to_glue_table_input_csv()
+            CuratedDataLoader(
+                column_metadata=schema_for_copy.data["TableInput"]["StorageDescriptor"][
+                    "Columns"
+                ],
+                table=QueryTable(
+                    database=schema_for_copy.database_name,
+                    name=schema_for_copy.table_name,
+                ),
+                table_path=self.new_curated_data_product_path.replace(
+                    self.schema.table_name, table
+                ),
+                athena_client=self.athena_client,
+                glue_client=self.glue_client,
+                logger=self.logger,
+            ).create_new_major_version_data_product_table(
+                is_updated_table=table == self.schema.table_name
```

Comment:
minor suggestion for readability

```suggestion
                is_updated_table=(table == self.schema.table_name)
```